src/dsp_tools/models/exceptions.py

```diff
@@ -83,25 +79,8 @@ class UserError(BaseError):
 
 
 class PermanentConnectionError(BaseError):
-    """
-    This error is raised when all attempts to reconnect to DSP have failed.
-
-    Attributes:
-        message: A message that describes the error
-    """
-
-    message: str
+    """This error is raised when all attempts to reconnect to DSP have failed."""
 
 
 class XmlUploadError(BaseError):
-    """
-    Represents an error raised in the context of the XML import.
-    """
-
-    _message: str
-
-    def __init__(self, msg: str):
-        self._message = msg
-
-    def __str__(self) -> str:
-        return f"XML-ERROR: {self._message}"
```

Comment:
This entire brimborium in unnecessary, because it inherits the attribute, the __init__ and the __str__ from BaseError. And since no other code depends on this specific string representation, it can be safely deleted.